CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Release History
+
+## 2.0.0 (2017-04-20)
+- Move Mixin and AMQP client to separate files
+- Replace AMQP connection handling code with latest internal version
+- Provide ability to regsiter callbacks for ready, unavailable, and persistent failure states
+- Provide mandatory AMQP properties (app_id, correlation_id, message_id, timestamp) automatically
+    - Mandatory properties cannot be overridden
+- Rename amqp_publish to publish_amqp_message (backwards-compatible shim is in place)
```

Comment:
Since this is a mixin and will be included in a request handler alongside other pieces of functionality, I would prefer to leave amqp in the signature to make the method easier to reason about at a glance.